app/src/main/java/dev/arkbuilders/rate/presentation/MainScreen.kt

```diff
@@ -36,13 +49,28 @@ fun MainScreen() {
         rootDefaultAnimations = RootNavGraphDefaultAnimations.ACCOMPANIST_FADING,
     )
     val navController = engine.rememberNavController()
+    val snackState = remember { SnackbarHostState() }
+
+    LaunchedEffect(key1 = Unit) {
+        DIManager.component.networkStatus().onlineStatus
+            .drop(1)
+            .collect { online ->
+                val visuals = if (online)
+                    ConnectivityOnlineSnackbarVisuals
+                else
+                    ConnectivityOfflineSnackbarVisuals
+                snackState.showSnackbar(visuals)
+            }
+    }
 
 
     val isKeyboardOpen by keyboardAsState()
     val bottomBarVisible = rememberSaveable { mutableStateOf(false) }
     val scope = rememberCoroutineScope()
```

Comment:
Please remove this as it's not used

Reply:
What exactly is not used? `scope`?

Reply:
Yeah. But as we are going to have klint later. It’s ok to leave it